ddtrace/tracer.py

```diff
@@ -200,6 +201,7 @@ def configure(
         dogstatsd_port=None,
         dogstatsd_url=None,
         writer=None,
+        queue_size=None,
```

Comment:
And I was trying to get rid of this `configure` thing :(

At least, add it to `__init__` also.

Reply:
is there any way we can add it to the global config instead?

```
from ddtrace import config
config.max_traces = 1000
```

Reply:
@jd: I know, I hate it :( but I didn't really see a way we can migrate when I wrote this... But now I see! I'll definitely add it to the __init__ so we're at least making some progress 😄 

@brettlangdon yes, also forgot about this.